src/components/services/PiHole.vue

```diff
@@ -49,7 +52,8 @@ export default {
   },
   methods: {
     fetchStatus: async function () {
-      const result = await this.fetch("/api.php").catch((e) => console.log(e));
+      const result = await this.fetch("/api.php${this.apiQuery}")
```

Comment:
Might require backticks for `${this.apiQuery}`.

Reply:
I believe it should be:

```js
const result = await this.fetch(`/api.php${this.apiQuery()}`)
```